docs/src/content/docs/id/guides/i18n.mdx

```diff
@@ -141,9 +141,12 @@ Jika terjemahan belum tersedia untuk suatu bahasa, Starlight akan menampilkan ko
 
 ## Terjemahkan UI Starlight
 
+import LanguagesList from '../../../../components/languages-list.astro';
+
 Selain menyimpan file konten yang diterjemahkan, Starlight memungkinkan Anda menerjemahkan string UI default (misalnya, judul "Di halaman ini" dalam daftar isi) sehingga pembaca Anda dapat mengakses website Anda sepenuhnya dalam bahasa yang dipilih.
 
-String UI yang diterjemahkan dalam bahasa Inggris, Ceko, Prancis, Jerman, Italia, Jepang, Portugis, Belanda, Denmark, Spanyol, Turki, Arab, Norwegia, Farsi, Ibrani, Cina (yang disederhanakan), Korea, Indonesia, Rusia, Swedia, Ukraina, Vietnam, dan Galisia sudah disediakan secara bawaan, dan kami menyambut [kontribusi untuk menambahkan lebih banyak bahasa default](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md).
+String UI yang diterjemahkan dalam bahasa <LanguagesList startsSentence /> sudah disediakan secara bawaan,
```

Comment:
Thanks @nazhard!

One small suggestion. Although not essential because the language names are capitalised anyway, but the `startsSentence` prop is intended for use when the component starts the sentence and should be capitalised:

```suggestion
String UI yang diterjemahkan dalam bahasa <LanguagesList /> sudah disediakan secara bawaan,
```